### PR TITLE
fixed(metrics): prometheus scrapping pod port

### PR DESCRIPTION
**Pull request by sergioalmendrosdiaz**
We were enabling port at service level and we really need the metrics from each pod

### CODE REVIEW

File: helm/thingsboard/values.yaml [hunk 1]
```diff
@@ -155,16 +155,13 @@ mqtt:
...
+    "prometheus.io/port": "8082"
     "prometheus.io/scrape": "true"
   podSecurityContext: {}
   securityContext: {}
   service:
     type: ClusterIP
-    http:
-      port: 8884
     port: 8883
-    additionalPorts: []
```
Comment (dmitry-midokura) [inline helm/thingsboard/values.yaml:167]: Is this related?

File: helm/thingsboard/values.yaml [hunk 2]
```diff
@@ -48,7 +48,7 @@ ingress:
 # endpoints in all services, refer to the Metrics
 # section in https://thingsboard.io/docs/user-guide/troubleshooting/
 monitoring:
-  enabled: false
```
Comment (ricardnacher) [inline helm/thingsboard/values.yaml:51]: IMO we should not enable them by default.

Comment (sergioalmendrosdiaz) [reply helm/thingsboard/values.yaml]: my fault, removed change

File: helm/thingsboard/templates/component-service.yaml [hunk 3]
```diff
@@ -44,12 +44,6 @@ spec:
       protocol: {{ $componentValues.port.protocol }}
       name: {{ $componentValues.port.name }}-{{ . }}
     {{- end -}}
-    {{- if $componentValues.http }}
```
Comment (dmitry-midokura) [inline helm/thingsboard/templates/component-service.yaml:47]: Needed?

Comment (sergioalmendrosdiaz) [reply helm/thingsboard/templates/component-service.yaml:47]: yes, we were exposing prometheus metrics in the load balancer, we don't need it for the prometheus scrapper

Comment (dmitry-midokura) [reply helm/thingsboard/templates/component-service.yaml:47]: Ok, I see more or less. Can we unset this in values.yaml for mqtt service?

Comment (sergioalmendrosdiaz) [reply helm/thingsboard/templates/component-service.yaml:47]: It was only added here by me for the mqtt service, all the other services don't have it and were working correctly before I added this part. It was a mistake on my part, it doesn't have any purpose anymore.

Comment (dmitry-midokura) [reply helm/thingsboard/templates/component-service.yaml:47]: Ok, thank you for the explanation.

**Review by dmitry-midokura — APPROVED** [review]
